model/project_ref.go

```diff
@@ -711,7 +711,7 @@ func (p *ProjectRef) DetachFromRepo(u *user.DBUser) error {
 	}
 	catcher.Add(UpsertAliasesForProject(repoAliasesToCopy, p.Id))
 
-	catcher.Add(GetAndLogProjectModified(p.Id, u.Id, false, before))
+	catcher.Add(GetAndLogProjectModifiedWithRepoAttachment(p.Id, u.Id, event.EventTypeProjectDetachedFromRepo, false, before))
```

Comment:
Can we add assertions that the event gets logged? I see there's already unit tests set up for detach/attach.